integration_tests/expr_01.py

```diff
@@ -4,13 +4,23 @@
 def add(x: i32, y: i32) -> i32:
     return x + y
 
+@inline
+def and_op(x: i32, y: i32) -> i32:
```

Comment:
I see. I think this feature isn't available in LLVM backend yet (will confirm though). I will add the same (after confirming).

cc: @certik @ronnuriel @rebcabin

Reply:
Go for it. Tests pass, but I assume LLVM ignores it.

Reply:
Well I am wondering if we should use the function inline pass for this. A function will be inlined in LLVM backend if it's decorated with `@inline`. Make sense?

In `--fast` flag all functions (provided they satisfy some conditions) are anyways inlined automatically.

Reply:
Yes. LLVM ignores this decorator. Making it work via function inline pass.